src/java/org/apache/cassandra/index/accord/RouteIndexFormat.java

```diff
@@ -178,7 +167,7 @@ public static ByteBuffer extractParticipants(RouteJournalIndex index, TxnId txnI
         }
     }
 
-    public static AccordJournal.Builder extract(TxnId txnId, ByteBuffer record, int user_version)
+    public static AccordJournal.Builder extract(TxnId txnId, ByteBuffer record, Version user_version)
```

Comment:
nit; pre-existing: Should we switch user_version to `userVersion` (i.e. camelCase)?